docs/public/installing-odo.adoc

```diff
@@ -46,28 +46,28 @@ The following section describes how to install `odo` on different platforms via
 
 === Binary installation
 
-. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo.exe`] file.
-. Add the location of your `odo.exe` to your `GOPATH/bin` directory.
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-windows-amd64.exe[`odo-windows-amd64.exe`] file.
+. Rename the downloaded file to `odo.exe` and move it to a folder of your choice, for example `C:\odo`
+. Add the location of your `odo.exe` to your `%PATH%`.
 
 === Setting the `PATH` variable for Windows 10
 
 Edit `Environment Variables` using search:
+The following examples demonstrates how to set up a path variable. Your binaries can be located in any location, but this example uses C:\odo as the location.
```

Comment:
1. It's got to be either `example demonstrates` or `examples demonstrate`. IMO, `examples demonstrates` is incorrect.
1. `Your binaries can be` - but in our example, we're covering only the odo binary, right? So maybe `s/binaries/binary`?

Reply:
Good catch, resolved.